pom.xml

```diff
@@ -171,8 +171,8 @@
             <artifactId>maven-compiler-plugin</artifactId>
             <version>3.1</version>
             <configuration>
-              <source>1.8</source>
-              <target>1.8</target>
+              <source>1.6</source>
```

Comment:
I guess it is forward compatible, but I think it would be best if we can compile against all 3 supported Java versions on travis